src/JuliusSweetland.OptiKey/Resources/Themes/Android_Dark.xaml

```diff
@@ -234,36 +238,16 @@
                         </MultiDataTrigger.Setters>
                     </MultiDataTrigger>
 
-                    <!--Show symbol if there is an image and the key is current-->
+                    <!-- ...or if Show Over Scratchpad-->
```

Comment:
Minor change as I'll forget what this means. Could we rename this property to "OnlyVisibleWhenInUse" or something similar?

Reply:
I've changed it to `OnlyVisibleWhenInUse`. I somehow missed this comment when I did the others earlier.